### PR TITLE
Add translation pipeline model

### PR DESCRIPTION
**Pull request by NickSchoelkopf**
add a translation pipeline model class (other lang -> translate to english -> summarization in english -> translate summaries to english) 

Addressing #109

### CODE REVIEW

**Comment by NickSchoelkopf** [comment]
still TODO: add this new model to the documentation

**Comment by NickSchoelkopf** [comment]
@niansong1996 should be ready for review now!

File: README.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ SummerTime supports different models (e.g., TextRank, BART, Longformer) as well
 | LongformerModel           | :heavy_check_mark:   |                      |                      |                      |                      |
 | MBartModel                | :heavy_check_mark:   |                      |                      |                      | 50 languages (Arabic, Czech, German, English, Spanish, Estonian, Finnish, French, Gujarati, Hindi, Italian, Japanese, Kazakh, Korean, Lithuanian, Latvian, Burmese, Nepali, Dutch, Romanian, Russian, Sinhala, Turkish, Vietnamese, Chinese, Afrikaans, Azerbaijani, Bengali, Persian, Hebrew, Croatian, Indonesian, Georgian, Khmer, Macedonian, Malayalam, Mongolian, Marathi, Polish, Pashto, Portuguese, Swedish, Tamil, Telugu, Thai, Tagalog, Ukrainian, Urdu, Xhosa, Slovenian) |
```
Comment (niansong1996) [inline README.md:88]: For the languages, maybe compile a list and only put a link here to that list?

File: summertime/model/single_doc/multilingual/translation_pipeline_model.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        self.assert_summ_input_type(corpus, queries)
+
+        src_lang = fasttext_predict(corpus)
+        # translate to English
+        corpus = self.translator.translate(
+            corpus, source_lang=src_lang, target_lang="en", beam_size=4
+        )
+        # TODO: translate each doc separately if provided multiple docs in corpus?
+        if queries:
+            queries = self.translator.translate(queries, target_lang="en", beam_size=4)
```
Comment (niansong1996) [inline summertime/model/single_doc/multilingual/translation_pipeline_model.py:121]: when you specify the beam_size as 4, is the output one sentence or 4?

**Review by niansong1996 — COMMENTED** [review]
Looks great! Left a few comments and should be ready to merge after you respond to them!

File: summertime/model/single_doc/multilingual/translation_pipeline_model.py (hunk 2 above)
Comment (NickSchoelkopf) [reply summertime/model/single_doc/multilingual/translation_pipeline_model.py:121]: Still just one sentence -- this is just the beam size for the translation model's generation.

**Comment by NickSchoelkopf** [comment]
All set with the changes! (assuming you were referring to the mBART-50 listed languages in the README)

**Comment by niansong1996** [comment]
LGTM, merging now

**Pull request merged**